### DISCUSSION TITLE
Is there an way to get a different {DATE} format in the folder structure.

### DISCUSSION THREAD

**Discussion by dantreacy** [Serene-Arc/bulk-downloader-for-reddit#636] [Q&A] [answered]
I'd like to have a folder for just the post year while keeping the default {date} format for use in the file scheme.. 

I've tried a few options but no success thus far. 

Is this possible whether using config or perhaps even some Python?

**Comment by Serene-Arc** [comment, marked as answer]
This is not a function of the BDFR, so you can't do it solely with this tool. However, it would be very simple to do with a bash script or something similar. Do you know how to use them?

**Reply by dantreacy** [reply]
Yeah I'll manage. Thanks. 

Consider this, if you would, a feature request :)

**Reply by Serene-Arc** [reply]
If you'd like a proper feature request, please open one as a bug report. I definitely won't remember to check this discussion
